lib/template.html

```diff
@@ -181,7 +181,7 @@
 {% endif %}
 
 {% if site.logo %}
-    "logo": {{ site.logo | prepend: "/" | prepend: seo_url | jsonify }},
+    "logo": {{ site.logo | prepend: seo_url | jsonify }},
```

Comment:
How many times do we use `site.logo`? Would it make sense to `assign` it at the top of the template?